src/app/workflow/view/view.component.html

```diff
@@ -17,24 +17,26 @@
 <button type="button" mat-button color="accent" class="btn-block" (click)="showVersionModal()">
   {{ isPublic || !canWrite || (entryType$ | async) === EntryType.Service ? 'View' : 'Edit' }}
 </button>
-<button
-  type="button"
-  mat-button
-  color="accent"
-  class="btn-block"
-  *ngIf="!version.frozen && version.referenceType !== 'BRANCH'"
-  (click)="snapshotVersion()"
-  data-cy="dockstore-snapshot"
->
-  Snapshot
-</button>
-<button
-  type="button"
-  mat-button
-  color="warn"
-  class="btn-block deleteVersionButton"
-  *ngIf="!isPublic && workflow?.mode === WorkflowType.ModeEnum.HOSTED && canWrite"
-  (click)="deleteHostedVersion(); (false)"
->
-  Delete
-</button>
+<ng-container *ngIf="!isPublic">
+  <button
+    type="button"
+    mat-button
+    color="accent"
+    class="btn-block"
+    *ngIf="!version.frozen && version.referenceType !== 'BRANCH'"
```

Comment:
Was like this, but I wonder if we need `version?...` since line 37 has `workflow?`.

Reply:
I think what should happen is that this component should be entirely hidden if there's no version